lib/jellyfish/room_service.ex

```diff
@@ -137,6 +170,34 @@ defmodule Jellyfish.RoomService do
     {:noreply, state}
   end
 
+  defp receive_resources(results \\ [], nodes \\ []) do
+    receive do
+      {:resources, node_name, resource_usage} ->
+        new_results = [resource_usage | results]
+        new_nodes = [node_name | nodes]
+
+        # Node.list() excludes local node
+        if Enum.count(new_nodes) == Enum.count(Node.list()) + 1 do
+          Enum.zip(new_nodes, new_results)
+        else
+          receive_resources(new_results, new_nodes)
+        end
+    after
+      1_000 ->
+        unknown_nodes = Node.list() |> Enum.reject(&(&1 in nodes))
+
+        Logger.warn(
+          "Nodes that don't respond on resource usage request #{inspect(unknown_nodes)}"
+        )
+
+        Enum.zip(nodes, results)
+    end
+  end
+
+  defp get_resource_usage() do
+    RoomService.list_rooms() |> Enum.count()
```

Comment:
It's probably going to change in the next PR, but the `RoomService.list_rooms()` function gets state of every room by making a blocking call to each of them. We only need the number of rooms so we could add a new function to `RoomService` which only gets the room ids from the registry. (without getting their state)

If you're going to change it anyway you can ignore this comment.

Reply:
Yup, plan is to change it in the following PR because the metric of the number of rooms is very naive.

Reply:
This change is made in #59